### ISSUE TITLE
Langpack in allowed_types is unclear

### ISSUE THREAD

**Issue by Eldertoon** [mozilla/policy-templates#682]
Hi,

My purpose: I want to enable 3 extensions through GPO to my users and any langpacks and dictionaries they want.

What I succeed: enabling 3 extensions with their IDs and block everything else.

What I am failing at: allowing langpacks and dictionaries for any language.

The documentation is unclear about allowed_types. It states that it whitelists the types listed. This is untrue for me as, if you write this as default:
`  "*": {
    "blocked_install_message": "restriction test 1",
    "install_sources": ["https://addons.mozilla.org/"],
    "installation_mode": "blocked",
    "allowed_types": ["extension"]
  },`
You still need to list one by one the extension that you want to allow.
-> In this example, langpack, dictionaries and theme will be forbidden even if you allow them (I suppose as I didn't test that far).
=> So this is not whitlisting things, that's just blacklisting other types?

For what I tested, it is not possible to have a wildcard in IDs like "langpack*", so I suppose it is not supported. The only workaround I can think of is to list every dictionaries and langpacks in the GPO.

Please tell me if I am correct

Thank you.

**Comment by Eldertoon** [comment]
Note: https://github.com/mozilla/policy-templates/issues/251 this issue seems to confirm what I told about blacklisting.

**Comment by mkaply** [comment]
Once you do blocked as the installation_mode, everything is blocked except what you specify explicitly. Not having extensions as a part of allowed_types accomplishes the same thing you want:
```
{
	"*": {
		"blocked_install_message": "restriction test 1",
		"install_sources": ["https://addons.mozilla.org/"],
		"allowed_types": ["dictionary", "langpack" ]
	},
        "foo@bar.com": {
            "installation_mode": "allowed"
        }
}
```
I'll double check this behavior against chrome, but recollection is I copied them directly. Basically, saying installation_mode blocked is a global switch that overrides allowed_types. Once I've tested, I'll update the documentation to make this more clear.

**Comment by Eldertoon** [comment]
Tested: it seems there is a bug. I first used `["langpack","dictionary"]` and dictionaries install worked fine but langpack still display a restriction error. Note that my previously installed langpack is still working, I could have expected it to be uninstall if I follow the logic of the documentation, shouldn't I?

I then used your writing (hoping it was just spacing issue) but it is still the same. Note that I am unable to install theme and extension (except for the one I allowed specificaly). I use Firefox 80.0.1 x64 if it helps.
